app/code/AKisilenko/ModuleLesson11/Block/View.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace AKisilenko\ModuleLesson11\Block;
+
+use AKisilenko\ModuleLesson11\Model\ConstructExample;
+use AKisilenko\ModuleLesson11\Model\FileList;
+use AKisilenko\ModuleLesson11\Model\ReflectionExample;
+
+/**
+ * Class View
+ * @package AKisilenko\ModuleLesson11\Block
+ */
+class View extends \Magento\Framework\View\Element\Template
+{
+    public $constructExampleModel;
```

Comment:
what is the reason of having this property public? Please share your oppinion. I would recommend making it private

Reply:
If my memory serves me, then I had an error and I searched for a reason. I was make it private